pulp_smash/tests/pulp2/rpm/api_v2/utils.py

```diff
@@ -418,6 +418,12 @@ def set_pulp_manage_rsync(cfg, boolean):
         client.run(sudo + ('which', 'setsebool'))
     except exceptions.CalledProcessError:
         return None
+
+    # setsebool may be available, but selinux could be disabled. Docker on OSX
+    enforcing = client.run(['getenforce']).stdout.strip()
+    if enforcing.lower() == 'disabled':
+        return None
```

Comment:
Beware that this new code substantially changes the behaviour of this function in ways that run contrary to the docstring. But see the other comment for more relevant suggestsions for improvement.